.travis.yml

```diff
@@ -31,8 +31,11 @@ cache:
   directories:
   - node_modules
 before_install:
-- wget https://github.com/bazelbuild/bazel/releases/download/0.11.0/bazel_0.11.0-linux-x86_64.deb
-- sudo dpkg -i bazel_0.11.0-linux-x86_64.deb
+- BAZEL_VERSION=0.11.0 \
```

Comment:
Would it be possible to use the `list` style yaml for these? Ie.
```
- BAZEL_VERSION=...
- wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/${BAZEL_DEB}
- sudo dpkg -i ${BAZEL_DEB}
- rm ${BAZEL_DEB}
```
It makes a bit easier to follow what is going on